### PR TITLE
proofread and edited pages folder

### PR DESCRIPTION
**Pull request by pau1tuck**
Proofread and edited two .md files in the pages directory:
- about.md
- contribution.md

### CODE REVIEW

File: pages/contribution.md [hunk 1]
```diff
@@ -51,8 +51,8 @@ If this is your first time contributing to the magazine, you should also add you
 
 ```diff
 [authors]
-rust-magazine = { name = "The Editor Team", editor = true, bio = "The magazine editor team" }
+rust-magazine = { name = "The Editorial Team", editor = true, bio = "The magazine editorial team" }
```
Comment (Folyd) [inline pages/contribution.md:54]: Good catch, I think we also should update the root `zine.toml` file.

Comment (pau1tuck) [reply pages/contribution.md:54]: Yes, good call.

File: pages/about.md [hunk 2]
```diff
@@ -19,21 +19,21 @@ Rust Magazine is maintained by Rustaceans from all around the world, currently c
...
 
 ## What is the release period of each issue?
 
-The frequency of issue release is one issue per month, each issue consisting of at most 10 articles.
+The frequency of issue release is one issue per month, with each issue consisting of, at most, 10 articles.
 
 ## Does the author get paid?
 
-While we are a non-profit community magazine and do not pay our authors, we deeply appreciate and value the contributions of our authors. We recognize the time and effort that goes into creating high-quality content, and we are grateful for the opportunity to feature the work of talented Rustaceans in our magazine.
+Although are a non-profit community magazine and do not pay our authors, we deeply appreciate and value their contributions. We recognize the time and effort that goes into creating high-quality content, and we are grateful for the opportunity to feature the work of talented Rustaceans in our magazine.
```
Comment (zephaniahong) [inline pages/about.md:36]: Although we

**Review by zephaniahong — CHANGES_REQUESTED** [review]

File: pages/about.md (hunk 2 above)
Comment (Folyd) [reply pages/about.md]: Good catch, thanks.

**Review by Folyd — APPROVED** [review]

**Pull request merged**